deploy/eck-stack/examples/custom-elasticsearch-kibana.yaml

```diff
@@ -0,0 +1,76 @@
+---
+eck-elasticsearch:
+  # Name of the Elasticsearch instance.
+  #
+  fullnameOverride: quickstart
+
+  # Version of Elasticsearch.
+  #
+  version: 8.2.3
+
+  nodeSets:
+  - name: default
+    count: 1
+    # Comment out when setting the vm.max_map_count via initContainer, as these are mutually exclusive.
+    # ref: https://www.elastic.co/guide/en/cloud-on-k8s/master/k8s-virtual-memory.html
+    #
+    config:
+      node.store.allow_mmap: false
+    volumeClaimTemplates:
+    - metadata:
+        name: elasticsearch-data
+      spec:
+        accessModes:
+        - ReadWriteOnce
+        resources:
+          requests:
+            storage: 100Gi
+        # Adjust to your storage class name
+        #
+        # storageClassName: local-storage
+
+eck-kibana:
+  # Name of the Kibana instance.
+  #
+  fullnameOverride: quickstart
+  
+  # Version of Kibana.
+  #
+  version: 8.2.3
+  
+  spec:
+    # Count of Kibana instances to create.
```

Comment:
See my comments with regards to `instance` above for the single product chart. Applies here as well.